hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,3 @@
+RELEASE_TYPE: patch
+
+Patch files written by hypothesis now use a deterministic ordering when multiple |@example| decorators are present.
```

Comment:
this is a nice thing to have, but the primary motivation for this patch is the hypofuzz use-case. (Also I haven't 100% confirmed we have order that depends on `PYTHONHASHSEED`, but it seems right.)